docs/en/operations/settings/settings.md

```diff
@@ -6,6 +6,39 @@ slug: /en/operations/settings/settings
 
 # Settings
 
+## additional_table_filters
+
+An additional filter expression that is applied after reading
```

Comment:
I'm not sure that `after reading` represents the idea. Nikolai wrote `will be applied directly after reading` and as I understand it means that we apply it before any other transformations/filters/etc are made.
@KochetovNicolai wdyt?

Reply:
thanks, I grabbed the `after reading` from the source (I think!), but happy to update it if needed.